content/docs/learning-guide/tutorials/multiplayer/first-multiplayer-component.md

```diff
@@ -910,3 +925,7 @@ Now that we have created a data flow from the server to clients using network pr
     </RemoteProcedure>
     </Component>
     ```
+
+10. At this point you should be able to see the client and server logs being emitted in the editor console when running the game in Play Mode:
+
+    ![Console logs emitted while running game from editor with RPC](/images/user-guide/multiplayer/add_myfirstnetworkcomponent_run_game_with_rpc.png)
```

Comment:
the doc's team might not like the hand drawn red arrow, they could probably clean that up

Reply:
ah, thanks for the heads up. I can remove that or make it look more polished.

Reply:
I recommend ShareX as a screenshot tool that includes features like drawing arrows. Here's some additional guidelines on [submitting media with image annotations](https://www.o3de.org/docs/contributing/to-docs/style-guide/media/#image-annotations).

Reply:
Its not 100% clear what the red arrow is pointing it at in the screenshot, it appears to be pointing at a specific line, where as I think you are trying to highlight the editor logs box

Reply:
Wouldn't be easy to just include the text of the log instead of a picture of a log?

Reply:
> Wouldn't be easier to just include the text of the log instead of a picture of a log?

I like the picture b/c it shows you where in the editor this would show up. Someone less familiar with the editor layout or running a game from it may not have a concrete idea for where to look.

Reply:
Maybe draw a red box around the log lines we want to highlight instead of using the red arrow? As Pip pointed out, the red arrow might be a little confusing since it seems pointing to a specific line.